src/Compilers/CSharp/Test/Emit2/Diagnostics/DiagnosticAnalyzerTests.cs

```diff
@@ -4360,5 +4360,40 @@ internal async Task TestMinimumReportedSeverity(SeverityFilter severityFilter, D
             Assert.True(analyzer.AnalyzerInvoked);
             Assert.Equal(expectedMinimumReportedSeverity, analyzer.MinimumReportedSeverity);
         }
+
+        [Fact, WorkItem("https://github.com/dotnet/roslyn/issues/74315")]
+        public async Task TestOperationConstructorBlockCallbackOnInvalidBaseCall()
+        {
+            // lang=C#-Test
```

Comment:
`lang=C#` turns on syntax highlighting in the string, right? but what does `lang=C#-Test` do?

Reply:
marks it as a C# test file, with supported C# test file syntax.  Note; we don't have `lang=c#` support (yet).  but it's something we could consider.